packages/palette/src/elements/Button/Button.shared.tsx

```diff
@@ -50,6 +50,8 @@ export interface ButtonBaseProps extends BoxProps {
   variantStyles?: any // FIXME
   /** Pass the longest text to the button for the button to keep longest text width */
   longestText?: string
+  /** The underlying type of button */
+  type?: string
```

Comment:
I'm not  sure if this is the right place for this. `ButtonBaseProps` is shared between web and native. Does RN have a `type` prop? If not, it might be best to move this to `Button.tsx`.

Reply:
Ok so I moved it from the shared file to the web one but then I had to name a new interface. I went with `WebButtonProps` but open to other ideas!